perf_workflow/run_perf_suite/test_suite.yaml

```diff
@@ -4,22 +4,26 @@
   Workload: eventdata
   DataNodes: 1
   MasterNodes: 0
+  use_50_percent_heap: "enable"
```

Comment:
Like discussed, let's remove the `use_50_percent_heap` from the config and set it to `enable` always.